rhel/Dockerfile

```diff
@@ -1,6 +1,4 @@
-ARG RHEL_VERSION=7
-
-FROM registry.access.redhat.com/ubi${RHEL_VERSION}/ubi
+FROM registry.access.redhat.com/ubi7/ubi:7.9@sha256:baccc7f4a9df7f4872116e254efb4e0e1b7d5d8e557947c02d44230ce7b905bc
```

Comment:
sha256 pins won't work for multi-architecture builds. I think this is currently fine just pointing it out for posterity / future us ;)

Reply:
Noted!